addons/account/views/account_analytic_line_views.xml

```diff
@@ -64,7 +64,7 @@
             <field name="inherit_id" ref="analytic.view_account_analytic_line_pivot"/>
             <field name="arch" type="xml">
                 <field name="account_id" position="after">
-                    <field name="partner_id" type="row"/>
+                    <field name="partner_id" type="row" invisible="context.get('group_by_date',False)"/>
```

Comment:
https://nimb.ws/mouz6O other costs/revenues links > I think we could group the graph, pivot, and grid views by date > month in the absence of the analytic account (do not set a default group by in the search view, it was just to show the result)
It was to complete this point. Since we group by default on the date, we didn't want to split it by partner. We should move that part in the view inherit of project though.